packages/terra-date-picker/src/clinical-lowlight-theme/DatePicker.module.scss

```diff
@@ -28,5 +28,9 @@
     --terra-date-picker-input-incomplete-disabled-border: 1px solid #181b1d;
     --terra-date-picker-input-incomplete-disabled-color: #464232;
     --terra-date-picker-input-incomplete-disabled-opacity: 0.3;
+
+    // Calendar-Filter
+    --terra-date-picker-calendar-filter-border: 1px solid #2d3539;
+    --terra-date-picker-calendar-filter-width: 16.78571rem;
```

Comment:
These should be added to their own theme file `packages/terra-date-picker/src/clinical-lowlight-theme/CalendarFilter.module.scss`

Reply:
Created CalendarFilter.module.scss for themes : [6d18c2607](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-2e6cd9f8dc05f96175347c56d18c2607)